### PR TITLE
You can no longer blind blind people

### LINKED ISSUE #43098: You can blind blind people with a flash
[Round ID]: 
I'll go one better and give you video evidence. https://youtu.be/S--3fMVBU7Q?t=585

## Testmerges: 
Not sure

## Reproduction: 
Give someone the blindness trauma and flash them.

Kinda makes no sense that you can blind a blinded person.

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #43098
:cl: ShizCalev
fix: You can no longer blind blind people with flashes.
/:cl:

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
eye open eye shut

**Comment by GuyonBroadway** [comment]
A thought occours, what about laser pointers and blinding blind people too?

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -296,6 +296,8 @@
 			return
 		if(visual)
 			return
+		if(!override_blindness_check && has_trait(TRAIT_BLIND))
```
[the remark this answers is not in the document]
Comment (Ghommie) [reply code/modules/mob/living/carbon/carbon_defense.dm]: Exactly what they said. This has to be moved above `(.)`.

**Comment by ShizCalev** [comment]
drunk pr

**Pull request closed without merging**